1) Maximum kredi limiti nedir? Hangi müşteriye ait?

2) Her ülkeden kaç müşteri var?

3) En az sipariş veren müşteri numarası ve sipariş sayısı nedir (orders tablosundan)?


In [2]:
import pandas as pd
import numpy as np
import warnings

2. Maximum kredi limiti nedir? Hangi müşteriye ait?

In [6]:
max_kredi_limit_musterisi = pd.read_csv("data/max_kredi_limit_musterisi.csv")
max_kredi_limit_musterisi

customerName  kredi_limit_max
0  Atelier graphique         227600.0

2. Her ülkeden kaç müşteri var?

In [5]:
ulke_basina_musteri = pd.read_csv("data/ulke_basina_musteri_sayisi.csv")
ulke_basina_musteri

country  count(*)
0         France        12
1            USA        36
2      Australia         5
3         Norway         3
4         Poland         1
5        Germany        13
6          Spain         7
7         Sweden         2
8        Denmark         2
9      Singapore         3
10      Portugal         2
11         Japan         2
12       Finland         3
13            UK         5
14       Ireland         2
15        Canada         3
16     Hong Kong         1
17         Italy         4
18   Switzerland         3
19   Netherlands         1
20       Belgium         2
21   New Zealand         4
22  South Africa         1
23       Austria         2
24   Philippines         1
25        Russia         1
26        Israel         1

3.  En az sipariş veren müşteri numarası ve sipariş sayısı nedir (orders tablosundan)?

In [7]:
en_az_siparis_veren_musteri = pd.read_csv('data/en_az_siparis_veren_musteri.csv')
en_az_siparis_veren_musteri

orderNumber  customerNumber  count(*)
0         10296             415         1
1         10228             173         2
2         10161             227         2
3         10191             259         2
4         10196             455         2
5         10206             202         2
6         10235             260         2
7         10242             456         2
8         10149             487         2
9         10210             177         2
10        10276             204         2
11        10222             239         2
12        10195             319         2
13        10183             339         2
14        10186             489         2
15        10232             240         2
16        10177             344         2
17        10207             495         2
18        10160             347         2
19        10187             211         2
20        10280             249         2
21        10285             286         2
22        10157             473         2
23        10180             171         2
24        10220             189         2
25        10225             298         2
26        10199             475         2
27        10154             219         2
28        10216             256         2
29        10284             299         2
..          ...             ...       ...
68        10194             146         3
69        10114             172         3
70        10130             198         3
71        10233             328         3
72        10202             357         3
73        10164             452         3
74        10101             128         4
75        10127             151         4
76        10272             157         4
77        10148             276         4
78        10107             131         4
79        10140             161         4
80        10116             381         4
81        10217             166         4
82        10139             282         4
83        10159             321         4
84        10119             382         4
85        10138             496         4
86        10112             144         4
87        10275             119         4
88        10250             450         4
89        10103             121         4
90        10258             398         4
91        10117             148         5
92        10120             114         5
93        10132             323         5
94        10105             145         5
95        10121             353         5
96        10113             124        17
97        10104             141        26

[98 rows x 3 columns]

En az siparis veren musteri numarasi 415 ve siparis adeti birdir. 